tests/ieee802154_hal/readme.md

```diff
@@ -0,0 +1,32 @@
+# Used Devices
```

Comment:
Probably we should keep this as a Tracking Issue in GH. So, we don't have to add PRs when this changes